### ISSUE TITLE
It should be possible to configure token to retrieve configuration from Consul

### ISSUE THREAD

**Issue by debae** [rikcarve/mp-config-consul#5]
Our Consul instance is protected by a token.
Within the extension however the client is called to retrieve key values without token.
```
...
  GetValue value = null;
  try {
      value = getClient().getKVValue(getPrefix() + key).getValue();
  } catch (Exception e) {
...
```

It would be nice if we could configure a token, and if that is provided fetch key values with it from consul.

**Comment by debae** [comment]
@rikcarve , do you accept a PR then I could pick this up if you like.
I would add a new optional property : **configsource.consul.token** and use that one if filled in.

**Comment by rikcarve** [comment]
Yes, sure, PR is welcome! Was on my list for later, but now it could be sooner :-)

**Comment by debae** [comment]
There you go : https://github.com/rikcarve/mp-config-consul/pull/6 , I've also bumped the version to 0.9 in the pom.xml

**Comment by rikcarve** [comment]
Cool! Thanks. Just 1 small thing, then I can merge and release it :-)

**Comment by rikcarve** [comment]
Fixed with #6

**Comment by rikcarve** [comment]
Version 0.9 will soon arrive on maven central
